docs/BLOCK_DEVICE_LOCKING.md

```diff
@@ -98,3 +98,145 @@ Summarizing: it is recommended to take `LOCK_EX` BSD file locks when
 manipulating block devices in all tools that change file system block devices
 (`mkfs`, `fsck`, …) or partition tables (`fdisk`, `parted`, …), right after
 opening the node.
+
+# Sample of Locking The Whole Disk
+
+The following is a sample to leverage `libsystemd` infrastructure to get the whole disk of a block device and add BSD lock on it.
```

Comment:
Not "sample", but "example".

"The following example leverages `libsystemd` to"

Also "take … lock" instead of "add … lock".